recipe/meta.yaml

```diff
@@ -108,10 +108,10 @@ outputs:
         # pytorch-cpu 1.1 which we built before conda-forge had GPU infrastructure
         # built into place.
         # https://github.com/conda-forge/pytorch-cpu-feedstock/issues/65
-        - pytorch-cpu = {{ version }}  # [cuda_compiler_version == "None"]
-        - pytorch-gpu = 99999999       # [cuda_compiler_version == "None"]
-        - pytorch-gpu = {{ version }}  # [cuda_compiler_version != "None"]
-        - pytorch-cpu = 99999999       # [cuda_compiler_version != "None"]
+        - {{ pin_subpackage('pytorch-cpu', exact=True) }}  # [cuda_compiler_version == "None"]
+        - pytorch-gpu ==99999999                           # [cuda_compiler_version == "None"]
+        - {{ pin_subpackage('pytorch-gpu', exact=True) }}  # [cuda_compiler_version != "None"]
+        - pytorch-cpu == 99999999                          # [cuda_compiler_version != "None"]
```

Comment:
```suggestion
        - {{ pin_subpackage('pytorch-cpu', exact=True) }}  # [cuda_compiler_version == "None"]
        - pytorch-gpu ==99999999                           # [cuda_compiler_version == "None"]
        - {{ pin_subpackage('pytorch-gpu', exact=True) }}  # [cuda_compiler_version != "None"]
        - pytorch-cpu ==99999999                          # [cuda_compiler_version != "None"]
```

Reply:
Missed a space maybe?

Reply:
i just don't think we can use pin_subpackage our recipe is weird.

Reply:
```
-   - package pytorch-2.0.0-cpu_generic_py39h7ecf1c4_1 has constraint pytorch-cpu 2.0.0 cpu_generic_py39h1234567_1 conflicting with pytorch-cpu-2.0.0-cpu_generic_py39hbe0b615_1
```